Sources/Filters/Sources/TriangleSource/index.js

```diff
@@ -0,0 +1,81 @@
+import macro from 'vtk.js/Sources/macro';
+import vtkPolyData from 'vtk.js/Sources/Common/DataModel/PolyData';
+import vtkMatrixBuilder from 'vtk.js/Sources/Common/Core/MatrixBuilder';
+
+// ----------------------------------------------------------------------------
+// vtkCustomTriangle methods
+// ----------------------------------------------------------------------------
+
+function vtkCustomTriangle(publicAPI, model) {
+  // Set our classname
+  model.classHierarchy.push('vtkCustomTriangle');
+
+  publicAPI.requestData = (inData, outData) => {
+    const dataset = vtkPolyData.newInstance();
+
+    const points = new Float32Array(3 * 3);
+    const edges = new Uint32Array(5);
+    edges[0] = 4;
+    edges[4] = 0;
```

Comment:
Not sure to understand why you have 5 edges ? Why the first one points to an unknown points ?

Reply:
Sounded weird for me too at first glance. I went from the star example in Filters/General/TriangleFilter which has the same pattern to define its edges. Since it worked I assumed this was intended.